app_legacy/Helpers/render/site-award.php

```diff
@@ -132,15 +136,20 @@ function RenderAwardGroup($awards, $title): void
         $counters = RenderCounter($icon, $text, $numItems, $numHidden);
     }
 
+    $fadeHiddenBottom = count($awards) <= 50 ? 'fade-hidden-bottom' : '';
     echo "<div id='" . strtolower(str_replace(' ', '', $title)) . "'>";
     echo "<h3 class='flex justify-between gap-2'><span class='grow'>$title</span>$counters</h3>";
-    echo "<div class='component flex flex-wrap justify-start gap-2'>";
+    echo "<div class='relative'>";
+    echo "<div onscroll='foo(event)' class='awards-fade fade-hidden-top $fadeHiddenBottom component flex flex-wrap justify-start gap-2 max-h-[340px] lg:max-h-[640px] lg:scrollbar-none overflow-y-auto transition'>";
```

Comment:
We should double check if the `transition` classname is actually needed here

Reply:
Also a good idea to move those dozens tailwind classes to `.awards-fade` in the CSS file.

Reply:
Personally, I still prefer dozens of tailwind classes over the @apply strategy as it requires building the frontend.
As discussed with @wescopeland there are SEO reasons for grouping css in classes, yet, I worked really hard to get rid of superfluous custom css classes in the past, I might be overly cautious with the recent trend of re-introducing classes.